.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release a new version
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  build:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create new production release
+      run: |
+        # Determine tag
+        base_tag="$(date +v%Y.%m.%d)"
+        tag="$base_tag"
+        # write last 30 tags to file. We shouldn't need to request further pages.
+        gh api "repos/${{ github.repository }}/tags" | jq -r '.[].name' > all_tags
```

Comment:
This looks to list the tags in reverse chronological order by default, but is it worth being explicit? I couldn't find docs (or figure out how to get the manpage from the gh cli) about the sort order.